USFMToolsSharp/Models/Markers/Marker.cs

```diff
@@ -28,15 +28,18 @@ public virtual string PreProcess(string input)
             return input;
         }
 
-        public bool TryInsert(Marker input)
+        public bool TryInsert(Marker input, bool checkPossibility = false)

```

Comment:
What is `checkPossibility` for?  It looks like maybe it's sort of a 'dry run' flag?